jetcd-core/src/main/java/io/etcd/jetcd/Response.java

```diff
@@ -37,13 +39,33 @@ interface Header {
          */
         long getClusterId();
 
+        /**
+         * Returns the cluster id.
+         * The value is actually an unsigned int64 value, which may exceed the range of long.
+         *
+         * @return unsigned cluster id
+         */
+        default BigInteger getUnsignedClusterId() {
+            return new BigInteger(Long.toUnsignedString(getClusterId()));
+        }
+
         /**
          * Returns the member id
          *
          * @return the member id.
          */
         long getMemberId();
 
+        /**
+         * Returns the member id.
+         * The value is actually an unsigned int64 value, which may exceed the range of long.
+         *
+         * @return unsigned member id
+         */
+        default BigInteger getUnsignedMemberId() {
+            return new BigInteger(Long.toUnsignedString(getMemberId()));
```

Comment:
Is there a way to avoid the intermediate string representation ? I'd like to avoid generating garbage if possible.